src/components/RadioInputBox/RadioInputBox.tsx

```diff
@@ -0,0 +1,53 @@
+import { useState } from "react";
+
+import type { ActionOptions } from "panels/ActionsPanel/ActionsPanel";
+
+import "./_radio-input-box.scss";
+
+type Props = {
+  name: string;
+  description: string;
+  options: ActionOptions;
+};
+
+export default function RadioInputBox({
+  name,
+  description,
+  options,
+}: Props): JSX.Element {
+  const [opened, setOpened] = useState<boolean>(false);
+
+  const handleSelect = (e: any) => {
+    const bool = e.target.value === "on" ? true : false;
+    setOpened(bool);
+  };
+  const labelId = `actionRadio-${name}`;
+
+  const generateOptions = (options: ActionOptions): JSX.Element[] => {
+    return options.map((option) => <input key={option.name} type="text" />);
+  };
+
+  return (
+    <div className="radio-input-box" aria-expanded={opened}>
+      <label className="p-radio u-no-padding--top radio-input-box__label">
```

Comment:
Sorry if you have a class now then you can remove the padding with that instead of the utility.